<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.990000000000009</double>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>W1</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.990000000000009</double>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>W2</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.990000000000009</double>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>W3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Dialog</receiver>
   <slot>setW1(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Dialog</receiver>
   <slot>setW2(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_3</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Dialog</receiver>
   <slot>setW3(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setW1(double)</slot>
  <slot>setW2(double)</slot>
  <slot>setW3(double)</slot>
 </slots>
</ui>
